Stylesheet/Components/badge.css

```diff
@@ -9,24 +9,26 @@
     position: absolute;
     z-index: 9999;
```

Comment:
giving the higher z-index value may cause problem

Reply:
Yes changing it